docs/config_entries_config_flow_handler.md

```diff
@@ -11,25 +14,25 @@ When instantiating the handler, Home Assistant will make sure to load all depend
 To register your config flow handler with Home Assistant, register it with the config entries `HANDLERS` registry:
 
 ```python
-from homeassistant import config_entries, data_entry_flow
+from homeassistant import config_entries
 
 @config_entries.HANDLERS.register(DOMAIN)
-class ExampleConfigFlow(data_entry_flow.FlowHandler):
+class ExampleConfigFlow(config_entries.ConfigFlow):
 ```
 
-> Temporarily, all config flow handlers will also need to add their component name to the `FLOWS` constant in `homeassistant/config_entries.py`. We are working on automating discovery.
+All config flow handlers will also need to add their domain name to the `FLOWS` constant in `homeassistant/config_entries.py`.
 
-## Initializing a config flow from an external source
+## Discovering your config flow
 
-You might want to initialize a config flow programmatically. For example, if we discover a device on the network that requires user interaction to finish setup. To do so, pass a source parameter and optional user input when initializing the config entry:
+Home Assistant has a discovery integration that scans the network for available devices and services  and will trigger the config flow of the appropriate integration. Discovery is limited to uPnP and zeroconf/mDNS.
 
-```python
-await hass.config_entries.flow.async_init(
-    'hue', data=discovery_info,
-    context={'source': config_entries.SOURCE_DISCOVERY})
-```
+To have your integration be discovered, you will have to extend the NetDisco library to be able to find your device. This is done by adding a new discoverable. [See the repository for examples of existing discoverable.](https://github.com/home-assistant/netdisco/tree/master/netdisco/discoverables)
```

Comment:
```suggestion
To have your integration be discovered, you will have to extend the [NetDisco library](https://github.com/home-assistant/netdisco) to be able to find your device. This is done by adding a new discoverable. [See the repository for examples of existing discoverable.](https://github.com/home-assistant/netdisco/tree/master/netdisco/discoverables)
```

Reply:
I know we link it a few words later but I think it's good to have a link to the repo itself as well.